packages/inngest/src/components/InngestCommHandler.ts

```diff
@@ -814,7 +814,7 @@ export class InngestCommHandler<
            * appropriate response is always given.
            */
           void actionRes.then((res) => {
-            return finalize(prepareActionRes(res));
+            void prepareActionRes(res).then(finalize);
```

Comment:
You knwo what I'm about to say!  Haha.  Can we add tests so that this won't regress? 🙏

Reply:
Added integration tests that use streaming for supported platforms. 👍